code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1671,50 +1671,123 @@
+						src.say_verb("The entrance to [station_name(1)].")
+						sleep(0.6 SECONDS)
+						src.say_verb("You will never find a more wretched hive of scum and villainy.")
+						sleep(0.7 SECONDS)
+						src.say_verb("I must be cautious.")
 					else if (istype(src.loc.loc, /area/station/engine/))
-						message = "<span style=\"color:black\"><B>[src]</B> says, \"The churning, hellish heart of the station that just can't help missing the beat. Full of the dregs of society, and not the right place to be caught unwanted. I better watch my back.\"</span>"
+						src.say_verb("The churning, hellish heart of the station that just can't help missing the beat.")
+						sleep(0.7 SECONDS)
+						src.say_verb("Full of the dregs of society, and not the right place to be caught unwanted.")
+						sleep(0.5 SECONDS)
+						src.say_verb("I better watch my back.")
 					else if (istype(src.loc.loc, /area/station/maintenance/disposal))
-						message = "<span style=\"color:black\"><B>[src]</B> says, \"Disposal. Usually bloodied, full of grey-suited corpses and broken windows. Down here, you can hear the quiet moaning of the station itself. It's like it's mourning. Mourning better days long gone, like assistants through these pipes.\"</span>"
+						src.say_verb("Disposal.")
+						sleep(1 SECONDS)
+						src.say_verb("Usually bloodied, full of grey-suited corpses and broken windows.")
+						sleep(0.6 SECONDS)
+						src.say_verb("Down here, you can hear the quiet moaning of the station itself.")
+						sleep(1 SECONDS)
+						src.say_verb("It's like it's mourning.")
+						sleep(0.6 SECONDS)
+						src.say_verb("Mourning better days long gone, like assistants through these pipes.")
 					else if (istype(src.loc.loc, /area/station/crew_quarters/cafeteria))
-						message = "<span style=\"color:black\"><B>[src]</B> says, \"A place to eat, but not an appealing one. I've heard rumours about this place, and if there's one thing I know, it's that it's not normal to eat people.\"</span>"
+						src.say_verb("A place to eat, but not an appealing one.")
+						sleep(0.6 SECONDS)
+						src.say_verb("I've heard rumours about this place, and if there's one thing I know, it's that it's not normal to eat people.")
 					else if (istype(src.wear_mask, /obj/item/clothing/mask/cigarette))
 						message = "<B>[src]</B> takes a drag on [his_or_her(src)] cigarette, surveying the scene around them carefullly."
 					else
-						message = "<B>[src]</B> looks uneasy, like [src.gender == MALE ? "" : "s"]he's missing a vital part of h[src.gender == MALE ? "im" : "er"]self. [src.gender == MALE ? "H" : "Sh"]e needs a smoke badly."
+						message = "<B>[src]</B> looks uneasy, like [hes_or_shes(src)] missing a vital part of [himself_or_herself(src)]. [capitalize(he_or_she(src))] needs a smoke badly."
```

Comment:
not going to split this to say_verb?

Reply:
Nah, this is an emote, rather than actual speech